### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by pavolloffay**
@sjoerdtalsma @objectiser could you please review?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
+
+# OpenTracing Instrumentation for `java.util.concurrent`
+OpenTracing instrumentation for `java.util.concurrent` package. In instruments `ExecutorService`,
```
Comment (sjoerdtalsma) [inline README.md]: Typo: "In instruments"-> "It instruments"?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+`Runnable` and `Callable`.
+
+## Configuration
+```java
+ExecutorService executorService = TracedExecutorService(Executors.newFixedThreadPool(4), tracer);
```
Comment (sjoerdtalsma) [inline README.md]: Missing `new` keyword?

**Review by sjoerdtalsma — COMMENTED** [review]
Looks good for a first version.
In a future version, I'd like to see a `CompletableFuture` implementation where followup-actions also run with the active span (e.g. `andThen` calls etc).
However, this is Java-8 only and certainly also has some caveats so probably best to save that for a future PR.

Thanks for picking this up so quickly @pavolloffay

**Comment by pavolloffay** [comment]
@sjoerdtalsma thanks for the review.

> However, this is Java-8 only and certainly also has some caveats so probably best to save that for a future PR.

It might be good to look at that too! :)

**Review by objectiser — APPROVED** [review]

**Comment by pavolloffay** [comment]
@objectiser thanks, I will merge and release

**Pull request merged**